docs/src/examples/components/Dialog/Content/index.tsx

```diff
@@ -15,6 +15,11 @@ const DialogContentExamples = () => (
       description="A dialog can contain an action in the header."
       examplePath="components/Dialog/Content/DialogExampleHeaderAction"
     />
+    <ComponentExample
+      title="Footer"
+      description="A dialog can contain a custom footer."
```

Comment:
```suggestion
      description="A dialog can contain a footer."
```

I want to keep this thing aligned with other examples.